tests/unit/eblob_wrapper.h

```diff
@@ -0,0 +1,120 @@
+#pragma once
+
+#include <boost/filesystem.hpp>
+
+#include <library/blob.h>
+
+#include "library/crypto/sha512.h"
+
+#include <eblob/eblob.hpp>
+
+#include <vector>
+
+
+class Item {
```

Comment:
Since the header became common, it's better to use more specific names. At least, to include `Item` as `eblob_wrapper` subtype.

Also the question about casing. In elliptics code style we use names `item` or `item_t` (underscore lowercasing)

Reply:
item_t looks good